README.md

```diff
@@ -34,6 +34,9 @@ Three schemas are used to define the metadata
 2. [closure schema](#closure-schema)
 3. [relationship schema](metadata-schema/relationshipSchema.schema.json)
 
+Additional schemas are used to define specific uses cases for metadata
```

Comment:
Rewording slightly `... specific uses cases for metadata` to `... specific use cases for metadata`